gordo_components/data_provider/ncs_reader.py

```diff
@@ -144,7 +145,10 @@ def read_tag_files(
         logger.info(f"Downloading tag: {tag} for years: {years}")
 
         for year in years:
-            file_path = tag_base_path + f"/{tag.name}/{tag.name}_{year}.csv"
+            tag_name_encoded = quote(tag.name, safe=" ")
+            file_path = (
+                tag_base_path + f"/{tag_name_encoded}/{tag_name_encoded}_{year}.csv"
```

Comment:
I see it was this way before, but maybe while you're at it, push in the `tag_base_path` into the string formatting as well?